src/rmrk1.0.0/classes/nft.ts

```diff
@@ -104,7 +104,7 @@ export class NFT {
     }
     return `${PREFIX}::${
       OP_TYPES.SEND
-    }::${VERSION}::${this.getId()}::${recipient}`;
+    }::${VERSION}::${this.getId()}::${recipient.replace(/\\s/g, "")}`;
```

Comment:
This is going to prevent space errors during generation of the interaction, but consolidator should deny SENDs that are invalid in this way.

Reply:
Yes there's validation in consolidator also

Reply:
https://github.com/rmrk-team/rmrk-tools/pull/49/files#diff-7af70d2402d6113195b3d72a780ac2b59efa5fbd2a563ec94a6581d413f40942R148